plugin/plugintest/api.go

```diff
@@ -1699,13 +1713,13 @@ func (_m *API) GetUserStatusesByIds(userIds []string) ([]*model.Status, *model.A
 	return r0, r1
 }
 
-// GetUsers provides a mock function with given fields: _a0
-func (_m *API) GetUsers(_a0 *model.UserGetOptions) ([]*model.User, *model.AppError) {
-	ret := _m.Called(_a0)
+// GetUsers provides a mock function with given fields: options
+func (_m *API) GetUsers(options *model.UserGetOptions) ([]*model.User, *model.AppError) {
```

Comment:
This file should have been updated in https://github.com/mattermost/mattermost-server/pull/10456, but didn't. Hence these changes are in this PR.